src/osfiber_windows.h

```diff
@@ -64,10 +64,12 @@ OSFiber::~OSFiber() {
 Allocator::unique_ptr<OSFiber> OSFiber::createFiberFromCurrentThread(
     Allocator* allocator) {
   auto out = allocator->make_unique<OSFiber>();
-  out->fiber = ConvertThreadToFiber(nullptr);
+  // Windows XP:  The FIBER_FLAG_FLOAT_SWITCH flag is not supported.
+  // ConvertThreadToFiber -->> ConvertThreadToFiberEx
```

Comment:
`// ConvertThreadToFiber -->> ConvertThreadToFiberEx`

This line is a bit meaningless. You've described a change that can be found in the git history.
I'd suggest removing this line.